ToDoApp/components/button/ButtonCustom.js

```diff
@@ -0,0 +1,30 @@
+import React, { Component } from 'react';
+import {Alert,TouchableHighlight, Text, View, StyleSheet} from 'react-native';
+ 
+class ButtonCustom extends Component {
+    render() {
+      return (
+        <View style={styles.container}>
+            <TouchableHighlight onPress={this.props._onPress} underlayColor="white">
+                <View style = {this.props.styleButton}>
+                  <Text style={styles.buttonText}>{this.props.text}</Text>
+                </View>
+            </TouchableHighlight>
+        </View>
+      );
+    }
+}
+
+module.exports = ButtonCustom;
```

Comment:
please use, `export const ButtomCustom`

Reply:
I have just changed that, in the next push it will be added.